include/LightGBM/config.h

```diff
@@ -859,6 +859,12 @@ struct Config {
   // desc = **Note**: can be used only in CLI version
   std::string output_result = "LightGBM_predict_result.txt";
 
+  // [no-save]
+  // desc = used only in ``prediction`` task
+  // desc = Split features of internal nodes that enable left-right random assignment mechnisim.
+  // desc = See section "A surrogate VIMP" of the paper https://doi.org/10.1214/07-EJS039 for more details.
+  std::vector<int> random_assign_features;
```

Comment:
Please make it clear in the documentation that this is a vector of feature *indexes* (integers containing the 0-based position of the features in the training data matrix) and not feature *names* (strings corresponding to column names).

Reply:
Thanks for your suggestion! I will modify the unclear description and fix the failure checks later.